main/acle.md

```diff
@@ -1072,6 +1083,31 @@ including the header:
 
 Including `<arm_sme.h>` also includes [`<arm_sve.h>`](#arm_sve.h).
 
+### Predefined feature macros and header files
+
```

Comment:
The goal here is to show that guarding  code with macros does not guarantee  a working binary, because this intrinsics may need more than that.

Reply:
The current description is better, but I wonder if it would be sufficient just to say: "No assumptions should be made on the order or context in which the preprocessor macros are evaluated, for example:"

What do you think @CarolineConcatto @rsandifo-arm  ?